modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFolderResourceImpl.java

```diff
@@ -136,11 +141,25 @@ public ObjectFolder postObjectFolder(ObjectFolder objectFolder)
 			throw new UnsupportedOperationException();
 		}
 
-		return _toObjectFolder(
+		com.liferay.object.model.ObjectFolder serviceBuilderObjectFolder =
 			_objectFolderService.addObjectFolder(
 				objectFolder.getExternalReferenceCode(),
 				LocalizedMapUtil.getLocalizedMap(objectFolder.getLabel()),
-				objectFolder.getName()));
+				objectFolder.getName());
+
+		return _toObjectFolder(
+			_objectFolderService.updateObjectFolder(
+				objectFolder.getExternalReferenceCode(),
+				serviceBuilderObjectFolder.getObjectFolderId(),
+				LocalizedMapUtil.getLocalizedMap(objectFolder.getLabel()),
+				_addOrUpdateObjectFolderItems(
+					serviceBuilderObjectFolder.getExternalReferenceCode(),
+					serviceBuilderObjectFolder.getObjectFolderId(),
+					ListUtil.filter(
```

Comment:
Do we need this filter?

Reply:
No, my mistake, forgot to remove it after the refactor